runner/runners/bazel.go

```diff
@@ -135,6 +136,22 @@ func fetchBazelCommandData(bzFiler *bzsnapshot.BzFiler, cmd *runner.Command, rts
 	}
 	cmd.Timeout = d
 
+	// Adjust environment as specified by supported platform properties
+	for _, platProp := range bzCommand.GetPlatform().GetProperties() {
+		if platProp.GetName() == "JDK_SYMLINK" {
+			javaHome, ok := os.LookupEnv("JAVA_HOME")
+			if !ok {
+				msg := "Unsuccessful lookup of $JAVA_HOME, not defined."
```

Comment:
Suggest more generic prefix for errors here - "Failed setting up platform property X: $reason"